tests/phpunit/tests/blocks/block.php

```diff
@@ -413,4 +413,158 @@ function test_passes_content_to_render_callback() {
 		$this->assertSame( 'abc', $block->render() );
 	}
 
+	/**
+	 * @ticket 53240
```

Comment:
I think 52991 might be a better ticket to reference here since that is what construct_wp_query_args was landed under

Reply:
@aaronjorbin, thank you for the feedback. It's very helpful 💯 I will apply the changes mentioned before committing.